packages/plugin-ext/src/plugin/type-converters.ts

```diff
@@ -949,6 +948,15 @@ export function getShellArgs(args: undefined | (string | theia.ShellQuotedString
     return result;
 }
 
+function getCommand(command: string | theia.ShellQuotedString): string {
+    if (typeof command === 'string') {
+        return command;
+    }
+    const shellQuotedCommand = command as theia.ShellQuotedString;
+
+    return shellQuotedCommand.value;
+}
```

Comment:
minor: can be simplified to:

```suggestion
function getCommand(command: string | theia.ShellQuotedString): string {
    return typeof command === 'string' ? command : command.value;
}
```

or 

```suggestion
function getCommand(command: string | theia.ShellQuotedString): string {
    if (typeof command === 'string') {
        return command;
    }
    return shellQuotedCommand.value;
}
```

Reply:
@vince-fugnitto thanks for the quick feedback. I will apply your comments.